website/content/docs/k8s/installation/vault/data-integration/webhook-certs.mdx

```diff
@@ -0,0 +1,261 @@
+---
+layout: docs
+page_title: Vault as the Webhook Certificate Provider for Consul Controller and Connect Inject on Kubernetes
+description: >-
+  Configuring the Consul Helm chart to use TLS certificates issued by Vault for the Consul Controller and Connect Inject webhooks.
+---
+
+# Vault as the Controller and Connect Inject Webhook Certificate Provider on Kubernetes
+## Prerequisites
+Prior to setting up this data integration between Vault and Consul on Kubernetes, you will need to:  
+1. Read and complete the steps in the [Systems Integration](/docs/k8s/installation/vault/systems-integration) section of [Vault as a Secrets Backend](/docs/k8s/installation/vault).
+1. Read the [Data Integration Overview](/docs/k8s/installation/vault/data-integration) section of [Vault as a Secrets Backend](/docs/k8s/installation/vault).
+1. Configure [Vault as the Server TLS Certificate Provider on Kubernetes](/docs/k8s/installation/vault/data-integration/server-tls) 
+1. Configure [Vault as the Service Mesh Certificate Provider on Kubernetes](/docs/k8s/installation/vault/data-integration/connect-ca)
+1. Complete the [Bootstrapping the PKI Engine for Controller and Connect Inject Webhooks](#bootstrapping-the-pki-engine-for-controller-and-connect-inject-webhooks) section.
+
+### Bootstrapping the PKI Engine for Controller and Connect Inject Webhooks
+
+First, we need to bootstrap the Vault cluster by enabling and configuring the PKI Secrets Engine to be able to serve TLS certificates for the controller and connect inject webhooks.
```

Comment:
```suggestion
The first step is to bootstrap the Vault cluster. Issue the following commands to enable and configure the PKI Secrets Engine to serve TLS certificates for the controller and connect inject webhooks:
```
"We" should only refer to HashiCorp. Address the reader as "you" or reword as an imperative so that "you" is implied.